nodeup/pkg/model/protokube.go

```diff
@@ -219,11 +220,23 @@ func (t *ProtokubeBuilder) ProtokubeFlags(k8sVersion semver.Version) *ProtokubeF
 	f := &ProtokubeFlags{
 		Channels:      t.NodeupConfig.Channels,
 		Containerized: fi.Bool(true),
-		EtcdImage:     s(fmt.Sprintf("gcr.io/google_containers/etcd:%s", imageVersion)),
 		LogLevel:      fi.Int32(4),
 		Master:        b(t.IsMaster),
 	}
 
+	// TODO this is dupicate code with etcd model
+	image := fmt.Sprintf("gcr.io/google_containers/etcd:%s", imageVersion)
+	assets := assets.NewAssetBuilder(nil)
+	remapped, err := assets.RemapImage(image)
+	if err != nil {
+		glog.Errorf("unable to remap container %q: %v", image, err)
```

Comment:
What is the reason to fall back?  Just return `fmt.Errorf` I think

Reply:
did not want to refactor method.  Keep the changes small.  I can refactor to return err

Reply:
refactored